src/main/java/hudson/plugins/sonar/SonarPlugin.java

```diff
@@ -34,7 +34,12 @@
 package hudson.plugins.sonar;
 
 import hudson.Plugin;
+import jenkins.model.Jenkins;
 
 public class SonarPlugin extends Plugin {
-
+  @Override
+  public void postInitialize() {
+    SonarGlobalConfiguration globalConfiguration = Jenkins.getInstance().getDescriptorByType(SonarGlobalConfiguration.class);
```

Comment:
![BLOCKER](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-blocker.png) NullPointerException might be thrown as 'getInstance' is nullable here [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS2259)